.circleci/config.yml

```diff
@@ -139,22 +139,36 @@ workflows:
             branches:
               ignore: master
 
-      - test_integration:
+      - app/test_system:
           name: dev-integration-tests-en
           context: org-global-v2
           working_directory: core
-          language: en
-          core_endpoint_url: https://${CIRCLE_PROJECT_REPONAME}-pr-${CIRCLE_PR_NUMBER}-core-en.dev.dialoguecorp.com
+          component: core-en
+          place: ca2
           requires:
             - deploy-dev-ca2-ephemeral
-      - test_integration:
+          stage: dev
+          steps:
+            - run-integration-tests:
+                language: en
+          filters:
+            branches:
+              ignore: master
```

Comment:
can you re-order the properties to make it homogeneous? it's a bit confusing right now.

Reply:
Changed it to be closer to the other jobs. Keep in mind it has several nested elements though.